python/tk-core/README.md

```diff
@@ -6,14 +6,14 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
-# ShotGrid Pipeline Toolkit Core API
+# Flow Production Tracking Toolkit Core API
```

Comment:
Please revert any change from inside the python/tk-core folder. This is a copy of the tk-core repository that will be updated when we release the equivalent changes in tk-core.